### ISSUE TITLE
This selector doesn't have any properties and won't be rendered...

### ISSUE THREAD

**Issue by kubajjz** [VineCode/Sass.novaextension#14]
Hi, can I change this behavior? I use some empty classes and tags and I dont want to delete  them.

It is more logical to skip those empty classes in final css, but keep them in original SASS file.
Thanks, Kubajjz

**Comment by vinecode-andy** [comment]
Hi @kubajjz  as far as I know this should be the default behaviour of the sass processor. If I create the following, the empty selectors are removed. 

**test.scss**
```
.test {}
.test-1 {}
.test-2 { color: red; }
```

**test.css** (Rendered)
```
.test-2 {
  color: red;
}
```

**Comment by kubajjz** [comment]
Yes, this is default behaviour of the sass processor, but when I have empty class and I tried to compilate it with your processor, I got error - warning: "This selector doesn't have any properties and won't be rendered".

it is annoying, getting error every time I save the file. Can I hide somehow?

![image](https://user-images.githubusercontent.com/30442118/101688279-16e64200-3a6c-11eb-9ae2-971ffee21555.png)
